vue/package.json

```diff
@@ -11,12 +11,12 @@
     "abp-web-resources": "^3.8.2",
     "axios": "^0.18.0",
     "famfamfam-flags": "^1.0.0",
-    "iview": "^3.0.0",
+    "iview": "^2.13.1",
```

Comment:
`~3.0.0` is fine. See https://github.com/aspnetboilerplate/module-zero-core-template/issues/343#issuecomment-437894647.

Reply:
Not release yet?

Reply:
Not changed yet, as I was hoping for a fix in iView.
You can modify your PR to make that change instead.
It's preferable as compared to downgrading a major version.

Remember to update that line in **yarn.lock**. Thanks!